docs/source/contributing.rst

```diff
@@ -282,6 +284,37 @@ See an example of how an index should look like:
     Note that in this examples we group ``audio_voice1`` and ``audio_voice2`` in a single clip because the annotation ``voice-f0`` annotation corresponds to their mixture. In contrast, the annotation ``voice-f0`` is extracted from the multiclip mix and it is stored in the ``multiclips`` group. The multiclip ``multiclip1`` has an additional clip ``multiclip1-mix.wav`` which may be the master clip, the final mix, the recording of ``multiclip1`` with another microphone.
 
 
+.. _upload_index:
+
+3. Uploading the index to an online repository
+----------------------------------------------
+
+To prevent an the exponential growth in size of the library, we are storing all indexes in an online repository, while removing these from the repository itself.
+For that task, we use Zenodo.
```

Comment:
```suggestion
To prevent an the exponential growth in size of the library, we are storing all indexes in an online repository, while removing these from the repository itself.
For that task, we use Zenodo.
```

This sentence seems to make sense only for already existing indices. I guess new ones are never added to the repo in the first place:

```suggestion
We store all indexes in an online repository on Zenodo.
```

Reply:
hey @hagenw, done! Thanks for the suggestion.